docs/tutorials/monero.md

```diff
@@ -0,0 +1,61 @@
+Monero
+==================
+
+## Connect to remote nodes.
+
+To connect Monero nodes, you have to configure the wallet software:
+
+In [Feather wallet](http://rwzulgcql2y3n6os2jhmhg6un2m33rylazfnzhf56likav47aylq.b32.i2p/): File -> Settings -> Network -> Proxy > i2p, with this option all network traffic will be routed over i2p, including websocket connection (if enabled).
+
+## Connection between other I2P nodes.
+
+1) Configure i2pd to create server type tunnel and put in your `./tunnels.conf` file:
+
+    [monero-node]
+    type = server
+    host = 127.0.0.1
+    port = 18080 # P2P port
+    keys = monero-node.dat
+
+2) Restart i2pd or send `SIGHUP` signal.
+
+3) Find your b32 address of the node.
+
+   Go to webconsole -> [I2P tunnels page](http://127.0.0.1:7070/?page=i2p_tunnels). Look for Server tunnels and you will see address like \<long random string\>.b32.i2p next to monero-node.
+
+   The node is now available in Invisible Internet. You can check the node's status by visiting \<long random string\>.b32.i2p/get_info.
+
+4) Update your monero.conf file;
+
+    anonymous-inbound=XXX.b32.i2p:18080,127.0.0.1:18080
+
+Replace XXX with your b32 address.
+
+
+## Setup I2P service on already running public node.
+
+Open wallet inteface (the "RPC") allows anyone to connect their wallets to Monero network through the node. This is useful for users who don't run their own nodes and connecting over i2p. 
+
+If you already have a node with public RPC and now want to run i2p service, follow those steps:
+
+1) Make sure the node is already configured for public use and running. (For an example you setted up `monero.conf` such as `public-node=1` with `restricted-rpc=1`)
+
+2) Configure i2pd to create server type tunnel and put in your `./tunnels.conf` file:
+
+    [monero-node-rpc]
+    type = server
+    host = 127.0.0.1
+    port = 18089 # Restricted RPC port
+    keys = monero-node.dat
+
+3) Restart i2pd or send `SIGHUP` signal.
+
+4) Find your b32 address of the node.
+
+   Go to webconsole -> [I2P tunnels page](http://127.0.0.1:7070/?page=i2p_tunnels). Look for Server tunnels and you will see address like \<long random string\>.b32.i2p next to monero-node.
```

Comment:
```suggestion
   You can view the RPC i2p.b32 address with:
`curl http://127.0.0.1:7070/?page=i2p_tunnels 2>&1 | grep -Eo "[a-zA-Z0-9./?=_%:-]*" | grep "18089"`
```

Reply:
Can be confusing for beginners.

Reply:
Its more confusing if you don't have a desktop environment. 

can add both instructions though?

Reply:
@nahuhh doesn't work, i2pd webconsole doesn't show the ports

Reply:
👀👀 it does on my system. Weird. Will check for  another method